### ISSUE TITLE
Wootomation stops working after locking and unlocking the PC

### ISSUE THREAD

**Issue by ItzNico** [WootingKb/wooting-macros#228]
**Describe the bug**
What title says.

**To Reproduce**
1. Open wootomation
2. Confirm it's working
3. Lock PC
4. Unlock PC
5. Try again

**Screenshots / Video**
[Here's a video replicating the bug.](https://youtu.be/5-1MGrmmLS0)
Note: Both keyboard and keypad "-" stop working as intended after unlocking the PC.

**Desktop (please complete the following information):**
 - OS: Windows 11

**Wootomation version**
Wootomation 1.1.0

**Comment by v-calvo** [comment]
I'm experiencing the same issue.

**Comment by Furby42** [comment]
I'm getting this same issue having just purchased the wooting 80he, would be great to get it addressed.

**Comment by Xantha123** [comment]
same problem here

**Comment by Ice2192** [comment]
2025 - Still have the same issue.

**Comment by v-calvo** [comment]
I'm not affiliated with this project in any way, but I think it's pretty safe to say that it is dead.
I recommend using the keyboard manager plugin that is build into Microsoft PowerToys.

**Comment by Ice2192** [comment]
To me it's just a small inconvenience. I just thought there would be some sort of solution to the problem. Besides, I need to assign a double mouse click to a button which MPT cannot do.

**Comment by Evan109** [comment]
killing the process from task manager and restarting it fixes the issue until the next time you lock your pc

**Comment by Ice2192** [comment]
Yeah I just quit via taskbar and reopen. Sucks the guy who worked on it left Wooting.

Edit: After a while (probably a lot longer than what most people can handle), I gave up on Wootomation and abandoned ship. Having to constantly open task manager to close the app and restart it got me very tired. I switched to AutoHotkey to map the double mouse click to F1 and then Microsoft PowerToys's Keyboard Manager to handle the other function mappings (F2-F24). To be honest, for a while now I've been using PowerToys and Wootomation at the same time and felt that in addition to Wootomation not working when unlocking my laptop, it has also been redundant when PowerToys also has a key remapper. I'm using an 80HE and used Wootility to make a second FN layer to make a second set of function rows using the current ones. Everything works after sleep and lockout. The responsiveness of the macros executing is unnoticeably different from Wootomation. I'm just leaving this here in case someone has the OP's issue as Wootomation is a lost cause for almost 2 years now and highly recommend NOT using it.
